lightspeed/modules/proc_generate-training-data-set.adoc 

```diff
@@ -0,0 +1,102 @@
+:_content-type: PROCEDURE
+
+[id="generate-training-data-set_{context}"]
+
+= Generating a custom model training data set
+
+After installing {AnsibleContentParser}, run it to scan your custom Ansible files and generate a custom model training data set. The training data set includes a list of Ansible files and their paths relative to the project root. You can then upload the  training data set to {ibmwatsonxcodeassistant} and create a custom model for your organization. If you used ansible-lint preprocessing and encountered rule violations, you must resolve the rule violations (URL) before uploading the training data set to {ibmwatsonxcodeassistant}.
+
+== Methods of generating a training data set
+You can generate a training data set by using one of the following methods:
+
+* *With ansible-lint preprocessing* 
++
+By default, the {AnsibleContentParser} generates training data sets by using ansible-lint preprocessing. The {AnsibleContentParser} uses ansible-lint rules to scan your Ansible files and ensure that the content adheres to Ansible best practices. If rule violations are found, the {AnsibleContentParser} excludes these files from the generated output. In such scenarios, you must resolve the rule violations, and run the {AnsibleContentParser} once again so that the generated output includes all your Ansible files. 
+
+* *Without ansible-lint preprocessing*
++
+You can specify to generate a training data set without ansible-lint preprocessing. In this method, the {AnsibleContentParser} does not scan your Ansible files for ansible-lint rule violations; therefore, the training data set includes all files. Although the training data set includes all files, it might not adhere to Ansible best practices and could affect the quality of your code recommendation experience. 
```

Comment:
Existing: "You can specify to generate a..."
Proposed: "You can generate a.."

Reply:
Done.